### PR TITLE
refactor(cc-input-text): use arrayEquals from utils

### LINKED ISSUE #563: cc-input-text: use arrayEquals from utils
We have:

* an `arrayEquals` inside the `cc-input-text.js` (not exported)
* an `arrayEquals` inside the `utils.js` (not used)

This function is duplicated in the codebase, we should make `cc-input-text.js` use the function from `utils.js`.

This way:

* the `arrayEquals` is type checked (because utils is the only file we type check for now)
* the `arrayEquals` could be unit tested later

### PR DESCRIPTION
**Pull request by HeleneAmouzou**
Fixes #563

### CODE REVIEW

**Review by hsablonniere — CHANGES_REQUESTED** [review]
Well done @HeleneAmouzou :clap: 

You will need to apply the lint command and rename your commit. I'll let @pdesoyres-cc guide you with this.

Apart from this, LGTM :+1:

**Review by florian-sanders-cc — APPROVED** [review]
Gg @HeleneAmouzou! 

Don't forget to update your commit message but apart from that it's all good! :wink:

**Review by Galimede — APPROVED** [review]
Apart from what has been said above, GG! :+1: :muscle:

**Comment by HeleneAmouzou** [comment]
> Well done @HeleneAmouzou 👏
> 
> You will need to apply the lint command and rename your commit. I'll let @pdesoyres-cc guide you with this.
> 
> Apart from this, LGTM 👍

File: src/components/cc-input-text/cc-input-text.js [hunk 1]
```diff
@@ -255,7 +234,8 @@ export class CcInputText extends LitElement {
           ` : ''}
         </label>
       ` : ''}
-
+      
+
```
Comment (roberttran-cc) [inline src/components/cc-input-text/cc-input-text.js:238]: **🤓 nitpick:** Those two empty lines has been committed mistakenly I guess (non blocking).

**Review by roberttran-cc — CHANGES_REQUESTED** [review]
Clean! It's always nice to remove some code, good job. :sunglasses: 

The commit message should be updated in my opinion (maybe it was a rebase glitch): `chore(cc-input-text): apply the lint command` states your last action when it should summarize the whole modification (like `chore(cc-input-text): replace inline function with the same from our internal utils` or something like that).

And also a little non-blocking feedback.

File: src/components/cc-input-text/cc-input-text.js (hunk 1 above)
Comment (HeleneAmouzou) [reply src/components/cc-input-text/cc-input-text.js]: Thanks, I have left one empty line as it was originally to espace the code :)

**Review by pdesoyres-cc — APPROVED** [review]
LGTM.

**Review by roberttran-cc — APPROVED** [review]

**Pull request merged**